### ISSUE TITLE
ambiguous use of `sha1`

### ISSUE THREAD

**Issue by SolaWing** [glock45/swifter#358]
the String extension `sha1` is conflict with [CryptoSwift](https://github.com/krzyzanowskim/CryptoSwift/blob/9284485ecd681b04d17e71383d903b5de33516db/Sources/CryptoSwift/String%2BExtension.swift#L26).

there are also many other names in extension may have a potencial collide with other's extension, like `toBase64`. so if these extensions only use internally, it can be marked as internal. or if need to use publicly, it should be added with prefix or namespace to avoid ambiguous

**Comment by adamkaplan** [comment]
Agreed, extensions should be marked internal unless they are needed for clients to use Swifter, and prefix makes sense for the properties and functions.

Though, the lack of prefix is as much a bug with CryptoSwift as it is with Swifter
